src/components/aeDivider/aeDivider.md

```diff
@@ -0,0 +1,5 @@
+Divider
```

Comment:
This shouldn't be necessary because `vue-styleguidist` already displays the name of a component.